packages/table/src/headers/_headers.scss

```diff
@@ -261,6 +261,9 @@ $dark-selectable-header-menu-selected-hover-background:
 
 .bp-table-row-headers .bp-table-header {
   min-width: $row-header-min-width;
+  // prevent text from leaking out of the cell during scroll, before the header
+  // width is updated on scroll end.
+  overflow: hidden;
```

Comment:
`#ThingsToWatchForAfterThisMerges`

 Do people put `inline` popovers or tooltips in their row headers? If so, this might break them. Fortunately, the context menu is _not_ `inline`, so that case is fine.

Reply:
Actually, these cells were already styled with `Classes.TABLE_TRUNCATED_TEXT`, which includes `overflow: hidden;`, so this is fine.

Reply:
I haven't seen `inline` popovers/tooltips in the wild, except with the legacy react-select stuff. Should be safe